### PR TITLE
all tests passing

### PR DESCRIPTION
**Pull request by tristntran**
Fixed the test. Please review and add comments

### CODE REVIEW

File: src/gamblers_ruin.py [hunk 1]
```diff
@@ -26,7 +26,7 @@ def find_nth_state(transition_matrix: np.ndarray,
     """
     # @ operator works for matrix multiplication in numpy
     # no need to cast to np.matrix
-    return np.linalg.matrix_power(transition_matrix, n) @ initial_state
+    return initial_state @ np.linalg.matrix_power(transition_matrix, n)
```
Comment (tristntran) [inline src/gamblers_ruin.py:29]: Why did we do this? I don't understand!!!

Comment (ProvenNoIQ) [reply src/gamblers_ruin.py:29]: Matrix multiplication is not commutative and the new one is the correct operation.

**Review by av-chapman — APPROVED** [review]
It's chillin, let's get this train wreck rollin finally!

**Review by devyn-miller — APPROVED** [review]
Cute

**Review by jordanbruce44 — APPROVED** [review]
LGTM

**Pull request merged**